manifest.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: matrixdeployment
+  name: matrixdeployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: matrixdeploymentpod
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: matrixdeploymentpod
+    spec:
+      containers:
+      - name: matrixcontainer
+        image: sbacr.azurecr.io/sbimage:latest
```

Comment:
It's probably a good idea to use specific tags to reference images in your manifests, otherwise your app could break unexpectedly, you don't need to change it now but keep it in mind.

Reply:
Noted. When I build a new image ill make sure to use a specific tag